run_openmm.py

```diff
@@ -24,6 +24,13 @@
 # Add a Monte Carlo barostat to the system for pressure control
 system.addForce(mm.MonteCarloBarostat(1*unit.atmospheres, 300*unit.kelvin, 25))
 
+custom_bond = mm.CustomBondForce("0.5*k*(r-r0)^2")
+custom_bond.addGlobalParameter("k", 100)
```

Comment:
When you write the function, you can pass an input parameter (for example, input_k), and replace the 100 with this variable.